test/addons-napi/test_array/test.js

```diff
@@ -19,19 +19,26 @@ const array = [
   ]
 ];
 
-assert.strictEqual(test_array.Test(array, array.length + 1),
+assert.strictEqual(test_array.TestGetElement(array, array.length + 1),
                    'Index out of bound!');
```

Comment:
Totally unrelated to this PR, but just wanted to say that this implementation doesn't look correct to me. "Index out of bound!" should be thrown as an Error. As it is, "Index out of bound!" could be a value in the Array.